ersilia/cli/commands/catalog.py

```diff
@@ -29,7 +31,23 @@ def catalog_cmd():
         default=False,
         help="Show more information than just the EOS identifier",
     )
-    def catalog(local=False, file_name=None, browser=False, more=False):
+    @click.option(
+        "--card",  # New --card option
+        type=click.STRING,
+        help="Display metadata for a specific model by providing its model ID",
+    )
+    def catalog(local=False, file_name=None, browser=False, more=False, card=None):
+        if card:
```

Comment:
When you make `card` variable a boolean type by configuring it as such, you should check for the model id through the `model` variable.